### ISSUE TITLE
Meteor Client crashes game[Bug] 

### ISSUE THREAD

**Issue by TheOnlySD12** [MeteorDevelopment/meteor-client#2374]
### Describe the bug

When I try to load the game it just crashes (it just closes, no fabric crash panel).
No other mods loaded, using TLaucher, tested without Meteor Client and it loads.
-Edit- Also, I installed other mods and all work (Sodium, Indium etc.)

[crash-2022-06-13_19.58.39-client.txt](https://github.com/MeteorDevelopment/meteor-client/files/8893112/crash-2022-06-13_19.58.39-client.txt)

### Steps to reproduce

```bash
1. Download TLauncher (If you don't have it)
2. Install Fabric 1.19 (From TLauncher version selector) - Game will automatically launch
3. Close game
4. Put Meteor Client in mods folder
5. Launch game
```

### Meteor Version

Release 0.4.9 & Dev Build 0.5.0 1495

### Minecraft Version

1.19

### Operating System

Windows

### Before submitting a bug report

- [x] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by Wide-Cat** [comment]
> TLauncher issue. We don't support cracked users/launchers.

typically yes, but this is a dev build issue

**Comment by TheOnlySD12** [comment]
> TLauncher issue. We don't support cracked users/launchers.

Actually, I tried it in 1.18.2 and it worked perfectly

**Comment by GoldenStack** [comment]
I get this while using the vanilla Minecraft launcher in 1.19.

**Comment by daedx** [comment]
Can confirm. The exit code is the same as the initial crash report provided by the OP. You can tell this isn't an exclusive T-Launcher issue. 

1.19, dev build 0.5.0 - 1495

**Comment by arlomcwalter** [comment]
fixed in 9e9a15f367b08727d2b5867ca4d7f8925a50c25d
